.idea/anti-racism-library.iml

```diff
@@ -0,0 +1,530 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure what this file is for, google tells me it's used for IDE configuration? Should this be in the .gitignore?

Reply:
Could it be from my RubyMine IDE?! I don't use the Cloud9 for now because I run out of space. I would add this file to .gitignore.